encode/encode.go

```diff
@@ -26,10 +26,11 @@ const (
 )
 
 type Screens struct {
-	roots  []render.Root
-	images []image.Image
-	delay  int32
-	MaxAge int32
+	roots             []render.Root
+	images            []image.Image
+	delay             int32
+	MaxAge            int32
+	ShowFullAnimation bool
```

Comment:
As a follow up to this change, we should update Pixlet to cut the animation at 15 seconds. If I'm developing using pixlet currently, the animation will run until completion. So making sure the behavior is: cuts off at 15 seconds when false, and runs until completion when true during development will be less confusing to folks.